### DISCUSSION TITLE
Multiple shortcuts for the same action

### DISCUSSION THREAD

**Discussion by davenicoll** [rxhanson/Rectangle#994] [Ideas]
When my mac is at home, I use the numeric keypad as shortcuts to positions on the screen which works great. When I'm on the road though, with just the built-in keyboard, I have to rebind the shortcuts. Any interest in adding an option to have 2 shortcuts for an action? Thanks.

**Comment by rxhanson** [comment]
Thanks for the feedback. Unfortunately, this isn't something I'm interested in adding to Rectangle.

As a workaround, you could use [Rectangle Pro](https://rectangleapp.com/pro) (paid) and implement custom shortcuts to duplicate the ones that you use, just with alternate shortcuts.
